plugins/jsontools/jsontools.plugin.zsh

```diff
@@ -11,9 +11,10 @@ if [[ $(whence node) != "" && ( "x$JSONTOOLS_METHOD" = "x"  || "x$JSONTOOLS_METH
 	alias urlencode_json='xargs -0 node -e "console.log(encodeURIComponent(process.argv[1]))"'
 	alias urldecode_json='xargs -0 node -e "console.log(decodeURIComponent(process.argv[1]))"'
 elif [[ $(whence python) != "" && ( "x$JSONTOOLS_METHOD" = "x" || "x$JSONTOOLS_METHOD" = "xpython" ) ]]; then
-	alias pp_json='python -mjson.tool'
+	alias pp_json='python3 -Im json.tool'
```

Comment:
If you don't want to use `-I`, the following would work:

```shell
alias pp_json="python -c 'import sys; sys.path.remove(\"\"); import runpy; runpy._run_module_as_main(\"json.tool\")'"
```

`runpy._run_module_as_main` doesn't appear to be documented but, if it were to prove to be a problem in the future, you could easily switch to `-I`.

Reply:
If you change the alias to `python3`, it will no longer match up with the `whence` invocation in the line above.

Reply:
Used your numpy suggestion. We can refactor the whole thing once Python 2 is dead.